react/features/base/logging/middleware.js

```diff
@@ -82,6 +87,16 @@ function _initLogging(loggingConfig) {
         Logger.addGlobalTransport(APP.logCollector);
         JitsiMeetJS.addGlobalLogTransport(APP.logCollector);
     }
+
+    if (isTestingEnabled) {
```

Comment:
This section should be brought into the if above. It should not execute on mobile currently. There's no such thing as global "APP" there.